README.md

```diff
@@ -1,2 +1,11 @@
-# Books
-Book example extension for extension developer
+# Books example
+
+This TYPO3 extension is an example for writing TYPO3 extensions.
+Is is used in the trainings provided by the TYPO3 Trainer Network.
+
+## License
+
+The code is licensed under the [Gnu General Public License V2](LICENSE).
+
+The extension icon is taken from [Font Awesome](https://fontawesome.com/)
+and is licensed under the [CC BY 4.0 License](https://fontawesome.com/license/free).
```

Comment:
Isn't it "CC-BY"?

Reply:
I'll look it up …

Reply:
According to https://creativecommons.org/licenses/by/4.0/deed.en, there is no hyphen.

Reply:
ah, there is a hyphen only if there are multiple limitations, e.g. in "CC BY-SA".